frame/balances/src/benchmarking.rs

```diff
@@ -33,7 +33,12 @@ const SEED: u32 = 0;
 // existential deposit multiplier
 const ED_MULTIPLIER: u32 = 10;
 
-benchmarks_instance_pallet! {
+impl_benchmark_test_suite!(
+	Balances,
+	crate::tests_composite::ExtBuilder::default().build(),
+	crate::tests_composite::Test,
+	by_instance_pallet,
+{
```

Comment:
Not a big fan of this specific syntax. I would prefer we keep the `benchmarks_instance_pallet!` top level macro, and instead have the change required by end users simply be to include the `impl_benchmark_test_suite` code block into the macro body.

does that make sense?

As in most people just need to move the bottom bracket to surround the old syntax, and indent it once.

Reply:
```
benchmarks_instance_pallet! {
    benchmark_1 { }: _() verify {}
    benchmark_2 { }: _() verify {}
    
    impl_benchmark_test_suite!(
	    Balances,
	    crate::tests_composite::ExtBuilder::default().build(),
	    crate::tests_composite::Test,
    );
}

Reply:
>I would prefer we keep the benchmarks_instance_pallet! top level macro, and instead have the change required by end users simply be to include the impl_benchmark_test_suite code block into the macro body.

So this would then be a breaking change, since the `impl_benchmark_test_suite!` macro cannot be used on its own anymore?  
@shawntabrizi

Reply:
@ucover This would not be a breaking change because your macro would simply parse this text, it wouldnt be a real macro.

Reply:
@thiolliere are there any tricks around that problem? ^

Reply:
I tried it with ` $(;)?` but that did not work. Yours works :thinking: